docs/source/boolean.rst

```diff
@@ -34,12 +34,18 @@ Yeast Networks
 
     .. autoattribute:: neet.boolean.examples.s_cerevisiae
         :annotation: = <neet.boolean.WTNetwork object>
-        
+
 p53 Networks
 """"""""""""""""""
 
     .. autoattribute:: neet.boolean.examples.p53_no_dmg
         :annotation: = <neet.boolean.WTNetwork object>
 
     .. autoattribute:: neet.boolean.examples.p53_dmg
-        :annotation: = <neet.boolean.WTNetwork object>        
+        :annotation: = <neet.boolean.WTNetwork object>
+
+
+Random networks
+^^^^^^^^^^^^^^^
+
+.. automodule:: neet.boolean.rannetwork
```

Comment:
Done. I am not sure if that is necessary but I made the indentation in the document consistently.